draft-ietf-acme-acme.md

```diff
@@ -955,16 +960,14 @@ invalid or unsupported contact URL, then the server MUST return an error of type
 "invalidContact", with a description describing the error and what types of
 contact URL the server considers acceptable.
 
-The server creates an account object with the included contact information.  The
-"key" element of the account is set to the public key used to verify the JWS
-(i.e., the "jwk" element of the JWS header).  The server returns this account
-object in a 201 (Created) response, with the account URI in a Location header
-field.
+The server populates the account object with the "key" field with the JWK public key
```

Comment:
How would you feel about `The server creates an account and populates the account object with ...`, I think it gets across that the server has to do something but keeps it a bit simpler.